README.md

```diff
@@ -1 +1,61 @@
-# Awsome-books
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome books
+
+>this project is for second module
```

Comment:
- [ ] please add A description that tells in detail what this project is about so others will know what is about